zerver/tests/test_auth_backends.py

```diff
@@ -2916,7 +2916,7 @@ def register_extra_endpoints(
             json=json.loads(settings.EXAMPLE_JWK),
         )
 
-    def generate_access_url_payload(self, account_data_dict: Dict[str, str]) -> str:
+    def generate_access_token_url_payload(self, account_data_dict: Dict[str, str]) -> str:
```

Comment:
It'd be nice to move these changes to the start of the PR so we can merge the refactors now.